Makefile

```diff
@@ -1,18 +1,17 @@
 HANDOFF_SPEC = HandoffSpecification
 DATABASE_SPEC = DatabaseSpecification
-HTML_DOCS = ../dist/$(DATABASE_SPEC).html ../dist/${HANDOFF_SPEC}.html
-TARGETS = ../docs/$(DATABASE_SPEC).adoc ../docs/${HANDOFF_SPEC}.adoc
+HTML_DOCS = ./dist/$(DATABASE_SPEC).html ./dist/${HANDOFF_SPEC}.html
+TARGETS = ./docs/$(DATABASE_SPEC).adoc ./docs/${HANDOFF_SPEC}.adoc
 # all: $(TARGETS)
+WETZEL = ./wetzel/bin/wetzel.js
 
-WETZEL = ../wetzel/bin/wetzel.js
+SCHEMALINK = ../src/data-validation/schema
```

Comment:
should this be single dot instead of double dot?

Reply:
Thanks for bringing it up, I've been experimenting with the dots quite a bit, and I found that a single dot in SCHEMALINK didn't work but using a double dot did

Reply:
Anyone seeing it might be suspicious, but I guess the program that is looking for it has a current working directory one level deeper.